build.sbt

```diff
@@ -4,40 +4,21 @@ import sbt._
 import Keys._
 import Defaults._
 
-organization := "io.swagger"
+organization := "__foursquare_shaded__.io.swagger"
 
-version := "1.0.7-SNAPSHOT"
+version := "1.0.6-fs0"
 
 scalaVersion := "2.11.12"
 
-crossScalaVersions := Seq("2.10.6", scalaVersion.value, "2.12.6", "2.13.1")
```

Comment:
oops, we should make sure we publish and upload this for scala 2.12 as well -- you can do that by adding this back and then adding a `+` to the sbt command: `+publishLocal`

Reply:
Added for scala 2.12 and 2.13